### PR TITLE
chore: bump rc-menu

### PR DESCRIPTION
**Pull request by Yuiai01**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [x] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  bump rc-menu          |
| 🇨🇳 Chinese | 更新 `rc-menu`           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -131,7 +131,7 @@
     "rc-input": "~0.2.1",
     "rc-input-number": "~7.4.0",
     "rc-mentions": "~2.1.0",
-    "rc-menu": "~9.8.2",
+    "rc-menu": "~9.8.3",
```
Comment (yoyo837) [inline package.json:134]: 这个不是必须的吧

Comment (yoyo837) [reply package.json:134]: 一遍只更新minor或者必要的最低版本依赖

**Comment by li-jia-nan** [comment]
patch 没锁的，会自动更新

File: package.json (hunk 1 above)
Comment (Yuiai01) [reply package.json:134]: 好~

**Comment by Yuiai01** [comment]
> patch 没锁的，会自动更新

好~  那我 close 掉吗

**Comment by yoyo837** [comment]
> > patch 没锁的，会自动更新
> 
> 好~ 那我 close 掉吗

合也无伤大雅. 只是不用专门来更新patch.

**Comment by Yuiai01** [comment]
> > > patch 没锁的，会自动更新
> > 
> > 
> > 好~ 那我 close 掉吗
> 
> 合也无伤大雅. 只是不用专门来更新patch.

好的，小弟记住了

File: package.json (hunk 1 above)
Comment (afc163) [reply package.json:134]: 升了也没关系，可以更新本地有锁的环境

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**